### DISCUSSION TITLE
Run Multiple Strimzi Versions in a single K8s Cluster

### DISCUSSION THREAD

**Discussion by Sai-Charan-Madhvaraj** [strimzi/strimzi-kafka-operator#9976] [Q&A] [answered]
Hi Team,

Do we have a way to run multiple strimzi versions in a single K8s cluster ?

We have a requirement to run the latest MM2 version 3.7.0 which is available in strimzi 0.40.0 but already there are Kafka Clusters runn ing with 3.4.0 with operator 0.35.1

Since the CRDs are cluster scoped on Kubernetes level , do we have any workaround to run 0.40.0 and 0.35.1 versions on same K8s cluster ?

**Comment by scholzj** [comment]
Did you look for the previous discussions and issues on this topic? I think this was quite extensively covered in the past.

**Reply by Sai-Charan-Madhvaraj** [reply]
Yes, I did and just want confirmation with new versions as there were bit old discussions.

**Reply by scholzj** [reply, marked as answer]
No, nothing changed recently. This is basically reflecting the Kubernetes design, so there is not much to improve.
